utilities_common/cli.py

```diff
@@ -200,6 +201,17 @@ def is_ipaddress(val):
         return False
     return True
 
+def ipaddress_type(val):
```

Comment:
is `ipaddress_type` used anywhere?

Reply:
Yes, on the other PR for the CLI:
https://github.com/Azure/sonic-buildimage/pull/8211
This is why this PR is a dependency for the other.